docs/news/posts/2024-10-18-exploring-aiida-database.md

```diff
@@ -0,0 +1,339 @@
+
+1. Load the node by using its UUID (note that the integer identifiers, called PKs, will most probably be different in your DB, but the UUIDs will always be the same):
+
+    ```python
+    In [1]: bandgap_node = load_node('138d1979-4aca-4a23-af18-38c707707585')
+    In [2]: bandgap_node
+    Out[2]: <Dict: uuid: 138d1979-4aca-4a23-af18-38c707707585 (pk: 692)>
+    ```
+
+2. One can check the attributes and discover that the band gap is in eV and is stored in the attribute named `band_gap` (knowing that the parser always returns eV, we are not going to use the units in the following, but one could generalise the query later if necessary). Note that the current node corresponds to a metal and we also see how this is represented.
+
+    ```python
+    In [3]: bandgap_node.attributes
+    Out[3]: {'is_insulator': False, 'band_gap': None, 'band_gap_units': 'eV'}
+    ```
+
+3. We start inspecting the provenance using the `.creator` method to get the calculation that generated the `Dict` data node:
+
+    ```python
+    In [4]: calculation_node = bandgap_node.creator
+    In [5]: calculation_node
+    Out[5]: <CalcFunctionNode: uuid: a2b360b9-4151-4917-b20d-6cde95ae5548 (pk: 45604)>
+    ```
+
+
+4. Then we can use `.inputs.LABEL_NAME` to access any of the inputs of this calculation. In our case, we will be interested in the one labeled `bands` (note that you can use tab completion to discover all input link labels: after `.inputs.` press the TAB key twice to see all available labels):
+
+    ```python
+    In [6]: bands_node = calculation_node.inputs.bands
+    In [7]: bands_node
+    Out[7]: <BandsData: uuid: 1b786a7c-639d-4fe4-a6e1-d0ac5b71bd39 (pk: 2829)>
+    ```
+
+5. In the same way we did before, we can now check the calculation that created this `BandsData` node (i.e., the band structure), to discover that it was a Quantum ESPRESSO run:
+
+    ```python
+    In [8]: qecalc_node = bands_node.creator
+    In [9]: qecalc_node
+    Out[9]: <CalcJobNode: uuid: c40cd8e3-638b-404e-b902-4377af810276 (pk: 18737) (aiida.calculations:quantumespresso.pw)>
+    ```
+
+6. Finally, we can check another input one level up to find the original crystal structure:
+
+    ```python
+    In [10]: qecalc_node.inputs.structure
+    Out[10]: <StructureData: uuid: 9e65c339-2428-4fb1-8d75-a54da9f87879 (pk: 43956)>
+    ```
+
+    Note that we don't really need all of the intermediate node variables.
+    We did it mostly for convenience, but all of these steps can just be concatenated in a single long chain, as follows:
+
+    ```python
+    In [11]: bandgap_node.creator.inputs.bands.creator.inputs.structure
+    Out[11]: <StructureData: uuid: 9e65c339-2428-4fb1-8d75-a54da9f87879 (pk: 43956)>
+    ```
+
+    The advantage of the long string above is that after every dot you can use tab completion, and therefore (once you get used to it) it becomes very quick to browse advanced provenance graphs in the verdi shell.
```

Comment:
```suggestion
    The advantage of the long string above is that after every dot you can use tab completion. Therefore, it can become very quick to browse advanced provenance graphs in the verdi shell and find the data you seek.
```